### PR TITLE
902719 - Prepare UG documentation for the Font autoscaling enabled feature. 

### PR DESCRIPTION
**Pull request by Arunachalam3641**
Hi @naveenkumar-sanjeevirayan,

I have prepared UG documentation for the Font autoscaling-enabled feature. 

Regards,
Arunachalam K

### CODE REVIEW

File: MAUI/Badge-View/Badge-customization.md [hunk 1]
```diff
@@ -344,3 +344,30 @@ Content = sfBadgeView;
 {% endtabs %}
 
 ![BadgeView Alignment](badge-customization_images/badge_alignment.png)
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Badge-View/Badge-customization.md:350]: The `FontAutoScalingEnabled` property is used to automatically scale the badge text's font size based on the operating system's text size. The default value of the FontAutoScalingEnabled property is false.

File: MAUI/Busy-Indicator/Set-Header.md [hunk 2]
```diff
@@ -199,3 +199,26 @@ The following screenshot illustrates the result of the above code.
 
 ![Font Customization](Images/SetHeader/Fontattributes.png)
 
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Busy-Indicator/Set-Header.md:204]: The `FontAutoScalingEnabled` property is used to automatically scale the .NET MAUI Busy Indicator's title font size based on the operating system's text size. The default value of the `FontAutoScalingEnabled` property is `false`.

File: MAUI/CheckBox/Visual-Customization.md [hunk 3]
```diff
@@ -261,5 +261,29 @@ The [`ControlSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.
     stackLayout.Children.Add(sfCheckBox);
     this.Content = stackLayout;
 
+{% endhighlight %}
+{% endtabs %}
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Visual-Customization.md:269]: The `FontAutoScalingEnabled` property is used to automatically scale the Radio button's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false`.

File: MAUI/Radial-Menu/SfRadialMenuItem-Customization.md [hunk 4]
```diff
@@ -729,3 +729,53 @@ namespace RadialMenuCommandSample
 }
 
 {% endhighlight %}
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/SfRadialMenuItem-Customization.md]: The `FontAutoScalingEnabled` property is used to automatically scale the Radial menu item's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false.`

File: MAUI/Radial-Menu/CenterButton-Customization.md [hunk 5]
```diff
@@ -918,3 +918,39 @@ namespace RadialSample
 {% endhighlight %}
 
 {% endtabs %}
+
+## CenterButtonFontAutoScalingEnabled
+
+The `CenterButtonFontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `CenterButtonFontAutoScalingEnabled` property is `false`. This property applies to both the  [`CenterButtonText`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadialMenu.html#Syncfusion_Maui_RadialMenu_SfRadialMenu_CenterButtonText) and the [`CenterButtonBackText`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.RadialMenu.SfRadialMenu.html#Syncfusion_Maui_RadialMenu_SfRadialMenu_CenterButtonBackText).
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radial-Menu/CenterButton-Customization.md:924]: The `CenterButtonFontAutoScalingEnabled` property is used to automatically scale the Radial menu's center button font size based on the operating system's text size. The default value of the `CenterButtonFontAutoScalingEnabled` property is `false.`

File: MAUI/TabView/Tab-Item-Customization.md [hunk 6]
```diff
@@ -478,4 +478,22 @@ tabView.IsScrollButtonEnabled = true;
 
 {% endtabs %}
 
-![TabView Scroll Mode](images/TabViewScroll.gif) 
\ No newline at end of file
+![TabView Scroll Mode](images/TabViewScroll.gif) 
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md]: The `FontAutoScalingEnabled` property is used to automatically scale the tab header's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false.`

File: MAUI/Badge-View/Badge-customization.md (hunk 1 above)
Comment (Arunachalam3641) [reply MAUI/Badge-View/Badge-customization.md]: Yes, changed.

File: MAUI/Busy-Indicator/Set-Header.md (hunk 2 above)
Comment (Arunachalam3641) [reply MAUI/Busy-Indicator/Set-Header.md]: Yes, changed.

File: MAUI/CheckBox/Visual-Customization.md (hunk 3 above)
Comment (Arunachalam3641) [reply MAUI/CheckBox/Visual-Customization.md]: Yes, changed.

File: MAUI/Radial-Menu/SfRadialMenuItem-Customization.md (hunk 4 above)
Comment (Arunachalam3641) [reply MAUI/Radial-Menu/SfRadialMenuItem-Customization.md]: Yes, changed.

File: MAUI/Radio-Button/Visual-Customization.md [hunk 7]
```diff
@@ -178,5 +178,29 @@ The [`ControlSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.
 	stackLayout.Children.Add(radioButton);
 	this.Content = stackLayout;
 
+{% endhighlight %}
+{% endtabs %}
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```
[the remark this answers is not in the document]
Comment (Arunachalam3641) [reply MAUI/Radio-Button/Visual-Customization.md]: Yes, changed.

File: MAUI/TabView/Tab-Item-Customization.md (hunk 6 above)
Comment (Arunachalam3641) [reply MAUI/TabView/Tab-Item-Customization.md]: Yes, changed.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**